nitrocli/src/commands.rs

```diff
@@ -625,10 +625,30 @@ fn choose_pin(pin_type: pinentry::PinType) -> Result<String> {
   }
 }
 
+/// Choose a PIN of the given type.
+///
+/// If the user has set the respective environment variable for the
+/// given PIN type, it will be used.
```

Comment:
We cannot read the new PIN from the same environment variable as the old PIN.  (While `pin_unblock` uses the current admin PIN to set a new user PIN, the `pin_set` command requires an old and a new PIN of the same type.)  We should have `NITROCLI_{ADMIN,USER}_PIN_NEW` variables instead.